libyul/backends/evm/EVMDialect.cpp

```diff
@@ -152,7 +158,7 @@ map<YulString, BuiltinFunctionForEVM> createBuiltins(langutil::EVMVersion _evmVe
 				_assembly.appendDataOffset(_context.subIDs.at(dataName));
 			}
 		}));
-		builtins.emplace(createFunction("datacopy", 3, 0, false, false, false, false, [](
+		builtins.emplace(createFunction("datacopy", 3, 0, false, false, false, false, true, false, [](
```

Comment:
```suggestion
		builtins.emplace(createFunction("datacopy", 3, 0, false, false, false, false, false, true, [](
```
?
The first one is storage and ``datacopy`` writes memory, doesn't it?
[side remark: Actually, why do we call it ``datacopy`` instead of ``codecopy``?]
And maybe we should have a struct with named members for all those flags...

Reply:
It should be fine. The last three flags are:
 - invalidates storage
 - invalidates memory
 - requires literal arguments

It is called datacopy instead of codecopy because we wanted to abstract away the fact that data is stored in code, since it might be different on other platforms.